src/main/groovy/org/jetbrains/intellij/tasks/RunIdeBase.groovy

```diff
@@ -236,7 +252,10 @@ abstract class RunIdeBase extends JavaExec {
         }
         systemPropertyIfNotDefined("idea.classpath.index.enabled", false, userDefinedSystemProperties)
         systemPropertyIfNotDefined("idea.is.internal", true, userDefinedSystemProperties)
-        systemPropertyIfNotDefined("idea.auto.reload.plugins", true, userDefinedSystemProperties)
+
+        if (!userDefinedSystemProperties.containsKey("idea.auto.reload.plugins") && isAutoReloadPlugins()) {
```

Comment:
shouldn't we set `idea.auto.reload.plugins=false` if system property wasn't set and autoReloadPlugins=false?

Reply:
If we pass the property, even as false, in 2020.1 and 2020.1.1 it will get enabled anyway (in IJ code there is check for property presence, not it's value), so not passing anything should fix the problem. This changes the behavior for newer versions as well, but I don't seen why would we want to pass the property as `false`?

Reply:
>  but I don't seen why would we want to pass the property as false?

It looks more consistent to me.

Ok, let's keep it in this way